tests/requirements-libraries.txt

```diff
@@ -54,6 +54,7 @@ Pillow==4.2.1
 pycryptodomex==3.4.6
 future==0.16.0
 pyusb==1.0.0
+requests==2.15.1 # Newest version of requests that allows tests to pass
```

Comment:
I don't understand the comment. Do we need 2.15.1 since older versions fail or since newer versions fail? 
Does this need to be pinned using a pyup-comment?